Signal/src/call/CallService.swift

```diff
@@ -608,7 +610,7 @@ fileprivate let timeoutSeconds = 60
         call.state = .connected
 
         // We don't risk transmitting any media until the remote client has admitted to being connected.
-        peerConnectionClient.setAudioEnabled(enabled: true)
+        peerConnectionClient.setAudioEnabled(enabled: !call.isMuted)
```

Comment:
Caller could mute the call before callee accepts call, so we should honor current model state.